jwst/coron/median_replace_img.py

```diff
@@ -81,6 +82,7 @@ def median_replace_img(img_model, box_size, bad_bitvalue):
     Returns
     -------
     img_model : image model
+
```

Comment:
The extra blank line is not needed here.

Reply:
Removed